tn_api_hotel.yml

```diff
@@ -551,7 +562,7 @@ components:
                   country:
                     type: string
                     description: The 2-letter country code for the reference point. Required for reference point.
-                    example: 'FR'
+                    example: 'FR'           
```

Comment:
Only thing I'd change is removing this extra whitespace at the end.

Reply:
will do! it must have snuck in.